src/api/type-defs/inputs/CreateInternalLabelsInput.ts

```diff
@@ -5,7 +5,7 @@ export default /* GraphQL */ `
     bbox: [Float!]!
     mlModel: String!
     mlModelVersion: String!
-    imageId: ID
+    imageId: ID!
```

Comment:
Required to make this work correctly:

https://github.com/tnc-ca-geo/animl-api/blob/76df33d98667de6bfdf837bb55fec9bbbf0226d9/src/api/db/models/Image.ts#L560-L578